tests/http_api.go

```diff
@@ -413,3 +417,26 @@ func (s *ClientFunctionalSuite) TestHTTPAPI_OperatorService_ListSearchAttributes
 	s.Require().Contains(searchAttrsResp.CustomAttributes, "CustomIntField")
 	s.Require().Equal(searchAttrsResp.CustomAttributes["CustomIntField"], "INDEXED_VALUE_TYPE_INT")
 }
+
+func (s *ClientFunctionalSuite) TestHTTPAPI_Serves_OpenAPIv2_Docs() {
+	_, respBody := s.httpGet(
+		http.StatusOK,
+		"/api/v1/swagger.json",
+		"",
+	)
+	s.T().Log(string(respBody))
```

Comment:
this is really big, right? maybe leave this commented out to reduce test output?

Reply:
I missed that one, thanks. I fixed it in the other function